packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -78,32 +125,85 @@ const styles = StyleSheet.create({
         top: 8,
     },
 
-    column: {
-        boxSizing: "border-box",
-        flex: "0 0 50%",
+    containerNoSidebar: {
+        minHeight: "auto",
+    },
 
-        // TODO(mdr): Some call sites probably won't want built-in padding. How
-        //     should we handle that possibility?
+    column: {
         paddingTop: 64,
         paddingBottom: 64,
+        minHeight: 464,
+        flex: "1 1 50%",
+        boxSizing: "border-box",
     },
 
-    leftColumn: {
+    bgSidebar: {
         background: Color.darkBlue,
         color: Color.white,
+    },
 
-        // TODO(mdr): Some call sites probably won't want built-in padding. How
-        //     should we handle that possibility?
-        paddingLeft: 64,
-        paddingRight: 52,
+    bgSidebarSmall: {
+        flex: "0",
     },
 
-    rightColumn: {
+    bgContents: {
         background: Color.white,
+        position: "relative",
+        display: "flex",
+        flexDirection: "column",
+    },
 
-        // TODO(mdr): Some call sites probably won't want built-in padding. How
-        //     should we handle that possibility?
-        paddingLeft: 52,
-        paddingRight: 64,
+    bgContentsNoSidebar: {
+        minHeight: "auto",
+        display: "block",
+    },
+
+    sidebar: {
+        marginRight: 32,
+    },
+
+    contents: {
+        marginLeft: 64,
+        overflow: "auto",
+        flex: "1 0 0",
+        maxWidth: 472,
+    },
+
+    contentsHasFooter: {
+        paddingBottom: 48,
+    },
+
+    contentsNoSidebar: {
+        marginLeft: 0,
+        maxWidth: "auto",
+    },
+
+    contentsSmall: {
+        margin: "0 auto",
```

Comment:
What does this do?

Reply:
In the "2-column with subordinated preview" (or in the implementation, when `smallSidebar` is `true`) the contents are essentially centered in the container - although I realize that I got the margins slightly wrong (as you noted where it dropped to 0) so I'll fix that.